src/providers/json-rpc-provider.ts

```diff
@@ -103,7 +111,7 @@ export class JsonRpcProvider extends Provider {
      * See [docs for more info](https://docs.nearprotocol.com/docs/interaction/rpc#validators)
      * @param blockId Block hash or height, or null for latest.
      */
-    async validators(blockId: BlockId): Promise<EpochValidatorInfo> {
+    async validators(blockId: BlockId | null): Promise<EpochValidatorInfo> {
```

Comment:
I think we should add `strictNullChecks` to our `tsconfig` to catch issues like this. @vgrichina do you have objections?

Reply:
Should be fine I guess, except seems like it might be quite a bit of work to convert existing codebase. 

Also I'd prefer we just make it optional `validators(blockId?: BlockId)` vs use `| null`.

Reply:
The RPC call [requires](https://docs.near.org/docs/interaction/rpc#validators) passing literal `null`:

    http post https://rpc.testnet.near.org jsonrpc=2.0 method=validators params:='[null]' id=dontcare